docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.7"
```

Comment:
issue: The default name for this file is now `compose.yml` or `compose.yaml`. I would change the name to match.
https://docs.docker.com/compose/compose-application-model/#the-compose-file